content/en/docs/reference/_index.md

```diff
@@ -79,6 +79,10 @@ operator to use or manage a cluster.
 * [Client authentication API (v1beta1)](/docs/reference/config-api/client-authentication.v1beta1/)
 * [WebhookAdmission configuration (v1)](/docs/reference/config-api/apiserver-webhookadmission.v1/)
 
+## Config API for kubeadm
+
+* [v1beta3](/docs/reference/config-api/kubeadm-config.v1beta3/)
```

Comment:
Hi @tengqm .
The changes look okay. Could the kubeadm configuration reference link move into the `Config APIs` list?
Possibly  something like,
`kubeadm v1beta3 configuration`?

Reply:
Yes, we can do that if desired.

Reply:
@kbhawkey Since https://github.com/kubernetes/website/pull/28871 is merged. We may want to postpone this consolidation of config APIs after the 1.22 is out? I don't want to cause many merge conflicts during this stage.